components/popover/index.js

```diff
@@ -213,6 +213,11 @@ class Popover extends Component {
 			isMobile,
 		} = this.state;
 
+		let { noArrow = false } = this.props;
+		if ( xAxis === 'center' && yAxis === 'middle' ) {
```

Comment:
Not certain about this? in which case will have both centered?

Reply:
Perhaps a `console.warn` would be better? I just want to explicitly declare somewhere that `position="middle center" noArrow={ false }` is something that the Popover component does **not** support.